README.rst

```diff
@@ -1164,6 +1164,17 @@ Choose from the list of available rules:
 
   *Risky rule: this fixer may change functions named setUp or tearDown outside of PHPUnit tests, when a class is wrongly seen as a PHPUnit test.*
 
+* **php_unit_static_assertion_calls**
+
+  PHPUnit assertion calls must all be of the same type.
+
+  *Risky rule: risky when PHPUnit methods are overridden or not accessible, or when project has PHPUnit incompatibilities.*
+
+  Configuration options:
+
+  - ``call_type`` (``'self'``, ``'static'``, ``'this'``): the call type to use for referring
+    to PHPUnit methods; defaults to ``'static'``
```

Comment:
please change default to `this`. i do know that official, syntax, method header is static, yet even creator is using `->` syntax, docs says so, it's most popular usage. doing anything else by default will be confusing for users

Reply:
I'll do so, but I hope it will change soon as static analysis tools get widespread and strict calls become mandatory.

Reply:
you remind me myself from 3 years ago when i was trying to persuade Sebastian ;)
in the meantime, it is like it is...

Reply:
*** it, let's be strict and spread the word. long live SCA.

Reply:
> (keep static as default)

I'm confused: do you want `$this` or `static` as default?

Reply:
you convinced me. static